### ISSUE TITLE
Update docs/getting-started.html to point to better content

### ISSUE THREAD

**Issue by kevmoo** [bp74/StageXL_Web#5]
The Dart Editor is no more.

Better to link to https://webdev.dartlang.org/guides/get-started

Cheers!

**Comment by bp74** [comment]
Yes you are right. The be honest i hate the whole StageXL website because it is so outdated.

**Comment by kevmoo** [comment]
You could always deprecate that site and set the homepage to be the repo – with a filled out README

Maybe?

**Comment by bp74** [comment]
Yes i though about that too. Also add a wiki to GitHub ... will think about that!
